frontend/packages/erd-core/src/features/erd/components/ERDContent/components/TableNode/TableColumnList/TableColumn/TableColumn.module.css

```diff
@@ -1,4 +1,7 @@
 .columnWrapper {
+  --bg-selected: #22392f;
+  --bg-hover: #434546;
```

Comment:
CSS Variables should be used as defined. (You should be able to retrieve the code from Figma.)